graftm/hmmer.py

```diff
@@ -414,7 +422,7 @@ def removeOverlaps(item):
                     continue
             return item
                     
-        def extractMultipleHits(reads_path, stats):
+        def extractMultipleHits(reads_path, stats): # TODO: Not used for the time being - need a more robust method
```

Comment:
Yes this was a regression that I forgot to fix up. I'll see if I can fix it before I submit another pull request